dashboard/app/models/user.rb

```diff
@@ -76,7 +76,7 @@
 
 class User < ActiveRecord::Base
   include SerializedProperties
-  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender)
+  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender races race_interstitial_shown)
```

Comment:
Perhaps this is overkill but how about when you select "Prefer not to say" all the other options are grayed out/disabled. You can only uncheck "Prefer not to say" to edit them again. Even if you submit with "Prefer not to say" checked and other options checked, we only submit "Prefer not to say" to the db.